lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php

```diff
@@ -34,7 +34,8 @@ protected function configure() : void
                 'write-sql',
                 null,
                 InputOption::VALUE_OPTIONAL,
-                'The path to output the migration SQL file instead of executing it. Default to current working directory.'
+                'The path to output the migration SQL file instead of executing it. Default to current working directory.',
```

Comment:
Should this be `Defaults to` or `Default is` instead?

Reply:
Updated to `Defaults`.